src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.html

```diff
@@ -229,6 +229,12 @@
             *ngIf="participationStatusWrapper() === ParticipationStatus.INACTIVE && isStartExerciseAvailable()"
             (click)="resumeProgrammingExercise()"
         ></button>
+        <button class="dev-button" id="submission" *ngIf="visibilityCheck()" (click)="simulateSubmission()">
```

Comment:
please only show these buttons in the exercise details page

Reply:
I moved the buttons, now they are only visible in the exercise detail page